### DISCUSSION TITLE
Welcome to UCanAccess discussions!

### DISCUSSION THREAD

**Discussion by spannm** [spannm/ucanaccess#8] [Announcements]
## 👋 Welcome!
  We’re using 'Discussions' as a place to connect with other members of our community. We hope that you:
  * Ask questions, provide answers and use cases
  * Share ideas and recommendations to move this project forward
  * Engage with other community members

Let's be nice, respectful and open-minded.
Thanks!

**Comment by yang351200** [comment]
May I ask if anyone can solve the data reading and writing problems in the 97 version of Access?
If possible, would it be convenient to share it?
